SparkPost/2web2ui
# SE-44: Ingest Batch Status

[SE-44](https://jira.int.messagesystems.com/browse/SE-44)

## What Changed

- New Signals Integration page

<img width="1389" alt="Screen Shot 2019-09-26 at 11 02 49 PM" src="https://user-images.githubusercontent.com/1335605/65739110-2f11b180-e0b2-11e9-94d5-eb2013c7dc8d.png">

## How to Test

- Enable "allow_events_ingest" account options
- Navigate to "/signals/integration" page
- Confirm filtering and pagination works as expected
